algorithms/glued_trees/glued_trees.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>In the background you mentioned the random labels of the nodes. It might be confusing as here there is no shuffling between the glued trees, and also the nodes are sorted. Give it an explanation so it will be clear </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I have added a paragraph explaining more clearly about how the matrix is modeling the columns of the glued trees system and therefore can avoid shuffling, etc. Additionally, I added a little more explanation about why -sqrt(A) is used as the Hamiltonian.